electron.vite.config.ts

```diff
@@ -66,6 +66,13 @@ export default defineConfig(({ mode }) => ({
       minify: true,
       sourcemap: mode === 'development' ? 'inline' : false
     },
+    css: {
+      preprocessorOptions: {
+        scss: {
+          api: 'modern'
+        }
+      }
+    },
```

Comment:
Is this required? According to [the docs](<https://vite.dev/config/shared-options#css-preprocessoroptions>), it should default to the `modern` API (we don't have `sass-embedded` installed; at least on `main` we don't

Reply:
it defaults to modern starting vite 6 afaik

Reply:
Note: We are on Vite 6 now; I'm fine with keeping this regardless, but might as well remove it if it's no longer required